carveracontroller/addons/pendant/pendant.py

```diff
@@ -207,7 +206,10 @@ def _handle_jogging(self, daemon: whb04.Daemon, steps: int) -> None:
                         else:
                             self._controller.startContinuousJog(f"{axis}{distance}", None, f"S{daemon.step_size_value}")
             else:
-                self._controller.jog(f"{axis}{distance}")
+                if daemon.step_size == whb04.StepSize.LEAD:
+                    self._controller.jog(f"{axis}{steps * 0.1}", abs(steps * 0.1 / 0.05) * 60 * 0.97)
```

Comment:
3 decimal places should be enough right?
```suggestion
                    self._controller.jog(f"{axis}{steps * 0.1}", round(abs(steps * 0.1 / 0.05) * 60 * 0.97), 3)
```

Reply:
Wouldn't it be better to just generally round the output commands to the machine in the jog method ?